src/test/java/com/cloudant/tests/ClientLoadTest.java

```diff
@@ -32,16 +32,16 @@
 	/** client max connections */
 	private static final int MAX_CONNECTIONS = 20;
 	
-	@BeforeClass 
-	public static void setUpClass() {
+	@Before
+	public  void setUpClass() {
```

Comment:
For this class it could be a `@Before`. The main reason why they should be `@Before` is so a failing test doesn't effect the next test. Since I'm generally unaware if a client has state left behind from each test. I thought it would be best to move them all to `@Before`. I'll make the method name change.